numpy/core/tests/test_multiarray.py

```diff
@@ -511,6 +515,23 @@ def test_array_as_keyword(self, func):
         else:
             func(a=3)
 
+    def test_scalar_and_wrongtype_intp_converter(self):
+        from numpy.core._multiarray_tests import test_scalar_intp_converter
+
+        assert test_scalar_intp_converter(10) == 10
+        # we also try -1 since it proved to be a problematic value
+        assert test_scalar_intp_converter(-1) == -1
+
+        assert_raises(TypeError, test_scalar_intp_converter, 'foo')
+
+    def test_scalar_and_wrongtype_intp_from_sequence(self):
+        from numpy.core._multiarray_tests import test_scalar_intp_from_sequence
+
+        assert test_scalar_intp_from_sequence(10) == 10
+        assert test_scalar_intp_from_sequence(-1) == -1
+
+        assert_raises(TypeError, test_scalar_intp_from_sequence, 'foo')
```

Comment:
Both of these tests look exactly the same, so you could move the import to the stop, and then rewrite them as:
```
@pytest.mark.parametrize("converter", [test_scalar_intp_converter, test_scalar_intp_from_sequence])
```
But I guess if you end up changing/extending, it is just as well to keep it as two.

Reply:
Sure, I'm just waiting to see the final form of `test_scalar_intp_converter` and `test_scalar_intp_from_sequence`